### ISSUE TITLE
How accurate is the conversion of text content into knowledge graphs based on large models?

### ISSUE THREAD

**Issue by PeterXiaTian** [microsoft/graphrag#449]
### Describe the issue

I dont know it .if we use general llm

### Additional Information

- GraphRAG Version:
- Operating System:
- Python Version:
- Related Issues:

**Comment by Shamepoo** [comment]
agree, the processing progress should be more visiable and analyzable to make adjustment decisions.
